src/riak_dt_lwwreg.erl

```diff
@@ -1,6 +1,6 @@
 %% -------------------------------------------------------------------
 %%
-%% riak_dt_lwwreg: A DVVSet based last-write-wins register
+%% riak_dt_lwwreg: A last-write-wins register
 %%
 %% Copyright (c) 2007-2013 Basho Technologies, Inc.  All Rights Reserved.
```

Comment:
you keep the 2013 here, whereas you moved to 2016 in riak_dt_gset?

Reply:
This was all done as part of the gset implementation, so no idea why one was changed but not the other.  I don't think there is any consistency over copyright notices on changes throughout the repos.  I don't know what the right answer is if define a new module, or modify an existing one.  Something to discuss with @martincox